APPX Generation very slow

### Environment

1. `react-native -v`: v0.59.10
2. `npm ls rnpm-plugin-windows`: 0.2.11
3. `npm ls react-native-windows`: 0.59.0-rc3
4. `node -v`: 10.15.0
5. `npm -v`: 6.4.1

Then, specify:

<!-- (What platform are you building for? Choose any from UWP, WPF) -->
- Target Platform: UWP

<!-- (What framework version are you building for? Choose from 10.0.14393, 10.0.10586, 10.0.10240, etc. (UWP), .NET 4.6.1, .NET 4.5, etc. (WPF) -->
- Target Platform Version(s): 10.0.16299.0

<!-- (What device(s) are you targeting? Choose any from Desktop, Mobile (ARM), Mobile Emulator, Xbox, Hololens) -->
- Target Device(s): Desktop

<!-- Which operating system are you using? Specify macOS, Windows, or Linux, along with specific release versions -->
- Development Operating System: Windows

<!-- Which version of Visual Studio are you using? Specify Visual Studio 2015 or Visual Studio 2017 -->
- Visual Studio Version: 2017 Community

<!-- Which build configuration are you running? Choose from Debug, DebugBundle, Release, ReleaseBundle -->
- Build Configuration: ReleaseBundle

### Steps to Reproduce

(Write your steps here:)

1. Attempt to generate the APPX bundle from either within VS or using msbuild on command line

### Expected Behavior

The APPX bundle would be created in a timely fashion.

### Actual Behavior

APPX bundle is taking in excess of an hour to be built. 

### Reproducible Demo

Repo available here: https://github.com/nelsona/windows-rn59-testapp

Running the following msbuild:
```
msbuild testApp.sln -t:restore /t:Clean;Build /t:testApp /p:OutputPath=./build;Configuration=ReleaseBundle;AppxBundle=Always;AppxBundlePlatforms="x64|ARM|x86";AppxPackageDir="build\\AppxPackages\\";UapAppxPackageBuildMode=StoreUpload
```
command shows the build getting stuck during the Processing application code section. Checking in task manager shows the .NET Native Compiler Toolchain process is running and doesn't seem to finish.

@licanhua This is working great for me if I use VS to build the APPX but when I try and build via MSBuild on the command line I get the following types of error in the the ReactNativeWebViewBridge project. It is complaining about not being able to find System so it reports errors like:
error CS0518: Predefined type 'System.Int32'

Can you give me any advice on how I might be able to get this running properly through the command line please?

@nelsona I used VS 2019 and v141 msbuild tool, I run into a different error: `LINK : fatal error C1001: An internal error has occurred in the compiler. [f:\repo\windows-rn59-testapp\node_modules\react-native-windows\Yoga\csharp\Yoga\Yoga.Universal.vcxproj]`
Add `/v:diag /bl` to msbuild will provide more logs for you. and you can open the binlog with http://msbuildlog.com/.

--- a/current/docs/SetupTroubleshooting.md
+++ b/current/docs/SetupTroubleshooting.md
@@ -81,3 +81,13 @@ Object reference not set to an instance of an object
 
 #### Solution:
 Install the 10.0.14393.0 Windows 10 SDK in the Visual Studio Installer
+
+#### Error:
+Appx generation with relese build takes a long time and freezes without completion.
+
+#### Solution:
+- Upgade Microsoft.NETCore.UniversalWindowsPlatform to 6.2.9
+- Move the minimum supported Windows version to 16299
+- Upgrade the react-native-windows plugin to 6.0.1
+
+See [Appx generation very slow issue](https://github.com/microsoft/react-native-windows/issues/3096) for detailed discussion.

